components/routing.rst

```diff
@@ -364,8 +364,7 @@ routes with UTF-8 characters:
 
     .. code-block:: php-annotations
 
-        // src/AppBundle/Controller/DefaultController.php
```

Comment:
Would it be possible to keep such comments? We can probably automatically just strip the AppBundle directory. The new one would be `src/Controller/DefaultController.php` in that case.

Reply:
Yes, I changed it everywhere, but we're in the components section, we shouldn't talk about framework conventions here. (probably, this needs to be extracted to its own subguide).

Reply:
ok, makes sense.